code/go/0chain.net/smartcontract/zcnsc/context_helpers_test.go

```diff
@@ -172,13 +147,13 @@ func MakeMockStateContext() *mockStateContext {
 	/// GetClientBalance
 
 	ctx.
-		On("GetClientBalance", mock.AnythingOfType("string")).
+		On("GetClientBalance", mock.AnythingOfType("string")). //nolint: typecheck
```

Comment:
Maybe we should replace `ctx.On` with ctx.StateContextI.On?  Or explicitly declaring
```go
var ctx  *mocks.StateContextI  := &mockStateContext{
  StateContextI: &mocks.StateContextI{},
}
```
Why are we wrapping the StateContextI anyway, bit confused.